Sources/TokamakCore/Views/TextField.swift

```diff
@@ -17,11 +17,9 @@
 
 public struct TextField<Label>: View where Label: View {
   let label: Label
-
-  // FIXME: this should be internal
-  public let textBinding: Binding<String>
-  public let editingChangedAction: (Bool) -> ()
-  public let commitAction: () -> ()
+  let textBinding: Binding<String>
+  let onEditingChanged: (Bool) -> ()
+  let onCommit: () -> ()
```

Comment:
@j-f1 you could the property here as you did in #125, but as internal, then add a corresponding computed property for it in the proxy, does that make sense?